BuildResidentialHPXML/tests/build_residential_hpxml_test.rb

```diff
@@ -134,6 +134,8 @@ def test_invalid_workflows
       'conditioned-attic-with-one-floor-above-grade.osw' => 'geometry_num_floors_above_grade=1 and geometry_attic_type=ConditionedAttic',
       'zero-number-of-bedrooms.osw' => 'geometry_num_bedrooms=0',
       'single-family-detached-with-shared-system.osw' => 'geometry_unit_type=single-family detached and heating_system_type=Shared Boiler w/ Baseboard',
+      'rim-joist-height-but-no-assembly-r.osw' => 'geometry_rim_joist_height=9.25 and rim_joist_assembly_r=false',
+      'rim-joist-assembly-r-but-no-height.osw' => 'rim_joist_assembly_r=23.0 and geometry_rim_joist_height=false',
```

Comment:
These are confusing error messages. Shouldn't it say something like, e.g., "... and rim_joist_assembly_r=not provided" instead of "... and rim_joist_assembly_r=false"? The current message makes it sound like a value of false was provided.

Reply:
I was just following the paradigm we've had up to this point. I can change it.